docs/source/ui/pages/run_management.md

```diff
@@ -189,3 +196,140 @@ Use the `Settings` tab to delete or archive the single run
+
+#### Statistics
+
+On this page, you can view the total number of runs, as well as the number of runs that are `archived`, `active`, and `finished`.
+You can click on the corresponding cards to view more information about these runs, and you can also use these cards to navigate to the [Runs explorer](../pages/run_management.html#runs-explorer) page with a specific query.
+The statistics bar, below the `status` cards, displays the distribution of runs by status.
+
+#### Contributions heatmap
+
+The heatmap shows the intensity of the runs (attached to the experiment), you have made for the day. More about Contributions heatmap you can read [here](../pages/home_page.html#id8).
+
+#### Activity feed
+
+This section represents the activity feed of your contributions related to the experiment. More about Activity feed you can read [here](../pages/home_page.html#id9).
+
+### Runs
+
+The Runs tab is for showing the list of runs attached to the experiment, it also gives ability to compare them by available explorers.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page runs tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-runs-tab.png"> 
+
+### Notes
+
+The Notes tab is for attaching simple text or rich Markdown as note to the experiment.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page notes tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-notes-tab.png">
+
+### Settings
+
+The Settings tab is for editing experiment name and description.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page settings tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-settings-tab.png">
+
+## Tags page
+
+### Overview
+Tags functionality intended to mark a runs. A tag can be attached to the runs to distribute by segments and then find it quickly.
+
+### Create tag
+How to create tag?
+There are two options for creating a tag.
+##### First option
+- Go to the tags page by clicking on the Tags from the left sidebar.
+
+- Click on the create tag button to open the create tag form modal. In this modal there is the form that has two input fields first one for tag name the second one for tag comment and also there are exist color pallets for selecting tag color.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Create tag" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/tags/1.png">
+
+- Type name for a tag. Name filed is mandatory and can't be empty for tag creation form and has maximum 50 symbol limit validation.
+- Type comment for a tag. Comment field is optional for tag creation form and has max 200 symbol limit validation.
```

Comment:
Type the tag description (?). The maximum number of characters is 200.

Reply:
Tags includes "comment" field on the Aim UI, if we want to change the word, we have to change both: from the UI and doc together.